docs/rfcs/2024-10-07-chat-infinite-scrolling.md

```diff
@@ -38,6 +38,15 @@ fetchLatest()
     fetchLast()
 ```
 
+
+```haskell
+-- Chat command
+APIGetInitialChatItems ChatRef
+
+-- Chat result
+CRInitialChatItems {user :: User, latestChatItems :: [AChatItem], unreadChatItems :: [AChatItem]}
```

Comment:
could also just be part of the API we currently use for loading a chat, if we can safely make breaking changes